gemdaq-testing/gemreadout/src/common/GEMDataParker.cc

```diff
@@ -151,44 +146,48 @@ uint32_t* gem::readout::GEMDataParker::getGLIBData(
       if (contqueue%kUPDATE7 == 0 &&  contqueue != 0) {
         contvfats_++;
         /*
-        INFO(" ::getGLIBData conter " << contqueue << " contvfats " << contvfats_ << " dataque.size " << dataque.size() 
+        INFO(" ::getGLIBData conter "  << contqueue << " contvfats " << contvfats_ << " dataque.size " << dataque.size() 
         */
       }
     }
 
-    uint32_t* pDQ = gem::readout::GEMDataParker::GEMEventMaker(bufferCount);
-    Counter[0] = *(pDQ+0);
-    Counter[2] = event_; // *(pDQ+2);
-    Counter[3] = *(pDQ+3);
+    uint32_t* pDQ = gem::readout::GEMDataParker::GEMEventMaker(Counter);
```

Comment:
I try put in other place but today GEMEventMaker works only here (+/-). 
If you know how activate access from GEMGLIBsupervisorWeb it'll very 
useful. I try but GEMEventMaker dosn't work properly.
    SB
>
> In gemdaq-testing/gemreadout/src/common/GEMDataParker.cc 
> <https://github.com/cms-gem-daq-project/gem-daq-code/pull/64#discussion_r41864139>:
>
> >          */
> >        }
> >      }
> >
> > -    uint32_t* pDQ = gem::readout::GEMDataParker::GEMEventMaker(bufferCount);
> > -    Counter[0] = *(pDQ+0);
> > -    Counter[2] = event_; // *(pDQ+2);
> > -    Counter[3] = *(pDQ+3);
> > +    uint32_t* pDQ = gem::readout::GEMDataParker::GEMEventMaker(Counter);
>
> This is still happening inside the "get data from GLIB" loop, which 
> means, that the loop won't continue until this function has returned.
> This is still a potentially massive bottleneck. I thought we discussed 
> how to improve this splitting
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-gem-daq-project/gem-daq-code/pull/64/files#r41864139>.
>